Feature/copy menu controller

igrekde opened the pull request:
This pull request provides the ability to copy the value from the selected token.

Full description:
When the user long presses the token a UIMenuController is shown with one action - copy. You can specify what exactly would be copied using the new method from the VENTokenFieldDataSource protocol. It is useful, for example, when your tokens represent contacts email-addresses, but the actual text is the contact's username.
Besides it, as we added the UILongPressGestureRecognizer to the VENToken, I decided to give the user opportunity to add its own longPressBlock, if for some reasons he doesn't want to use the menu controller functionality.
And, of course, I've updated the README file.

igrekde commented:
@ayanonagon, Hi, can you have a look at this PR? :)

ayanonagon commented:
Thank you for the PR! This is cool. Do you think the delegate method might be more clearly named as `tokenField:copyStringForTokenAtIndex:`? Also if the delegate does not implement that method, a sensible default might be the title of the token. Let me know what you think. :octocat:

igrekde commented:
@ayanonagon Hi! Thanks for watching my PR. I've already tried to name this method as **copyString:**, but there was a problem with **VENToken** interface - it cannot have a **copyString** property, because **copy** a reserved system prefix.
And yes, it's a good idea to use the title as the default value - I'll add this functionality.

igrekde commented:
@ayanonagon I've added the default value for copying and renamed all methods and properties from **underlyingString** to **copyString**. Have a look please.

ayanonagon commented on VENTokenField/VENToken.m:
```diff
@@ -57,6 +61,11 @@ - (void)setTitleText:(NSString *)text
 
+- (void)setTokenCopyString:(NSString *)string
+{
+    self.tokenCopyString = string;
```
This will cause an infinite loop! :recycle:

ayanonagon commented:
Hi, sorry about the delay. Finally getting to catch up on some PR’s. I just tried this out and got stuck in an infinite loop.

Thanks for the PR! :octocat:
